File: app/src/main/java/com/smart/resources/schools_app/features/advertising/AdvertisingModel.java
package com.smart.resources.schools_app.features.advertising;

import java.io.Serializable;
import org.threeten.bp.LocalDateTime;

public class AdvertisingModel implements Serializable {
	private int advertisementId;
	private String title;
	private String attachment;
	private String note;
	private LocalDateTime date;

	public int getAdvertisementId(){
		return advertisementId;
	}

	public String getTitle(){
		return title;
	}

	public String getAttachment(){
		return attachment;
	}

	public String getNote(){
		return note;
	}

	public LocalDateTime getDate(){
		return date;
	}


}

File: app/src/main/java/com/smart/resources/schools_app/features/advertising/AdvertisingModel.kt
package com.smart.resources.schools_app.features.advertising

import com.smart.resources.schools_app.core.extentions.isImage
import org.threeten.bp.LocalDate

data class AdvertisingModel(
    val advertisementId: String,
    val title: String,
    val attachment: String,
    val note: String,
    val date: LocalDate
) {
    val hasImage get() = attachment.isImage()
}